conda_build/scripts.py

```diff
@@ -37,7 +37,7 @@ def iter_entry_points(items):
 
 
 def create_entry_point(path, module, func):
-    pyscript = PY_TMPL % (module, func, func)
+    pyscript = PY_TMPL % {'module': module, 'func': func}
```

Comment:
@pelson Hmm, how about "this helps to avoid possible mistakes and improve readability"?